_maps/map_catalogue.txt

```diff
@@ -531,3 +531,45 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 35)(y = 21)(z = 1)
 		Tags = "No Combat", "Medium Loot", "Shelter"
 
+
+		Waste Ruins:
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_clowncrash.dmm"
+		Size = (x = 11)(y = 12)(z = 1)
+		Tags = "No Combat", "Minor Loot", "Shelter" "hospitable"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_crashed_kitchen.dmm"
+		Size = (x = 11)(y = 16)(z = 1)
+		Tags = "Minor Combat Challenge", "Medium Loot", "hospitable"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_fortress_of_solitude.dmm"
+		Size = (x = 45)(y = 42)(z = 1)
+		Tags =  "Minor Combat Challenge" "Medium Loot" "hospitable" "shelter"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_lab.dmm"
+		Size = (x = 26)(y = 25)(z = 1)
+		Tags = "No Combat", "Medium Loot" "Shelter" "hazardous"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_ore_processing_facility.dmm"
+		Size = (x = 30)(y = 26)(z = 1)
+		Tags = "No Combat", "Medium Loot" "Hazardous" "Lava"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_pandora.dmm"
+		Size = (x = 18)(y = 21)(z = 1)
+		Tags = "Boss Combat Challenge", "Minor Loot" "Megafauna", "Shelter"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_pod.dmm"
+		Size = (x = 8)(y = 8)(z = 1)
+		Tags = "Medium Combat Challenge", "Medium Loot"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_tarpit.dmm"
+		Size = (x = 30)(y = 30)(z = 1)
+		Tags = "No Combat", "Medium Loot", "hospitable"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_tradepost.dmm"
+		Size = (x = 15)(y = 20)(z = 1)
+		Tags "No combat", "Medium loot", "shelter"
+
+		File name ="_maps\RandomRuins\wasteruins\wasteplanet_unhonorable.dmm"
+		Size = (x = 11)(y = 17)(z = 1)
+		Tags = "No Combat", "Medium Loot", "Shelter", "Hazardous"
```

Comment:
Just to make sure you're using hazardous right.
Hazardous is saying there are map hazards that aren't lava or one of its reskins (acid and liquid plasma) which are tagged as "lava". Hazardous can be used for: active bear traps, IEDs, crushers, wizard spell traps, and other various hazards.

Reply:
Yep. Unhonorable has a radiation hazard to it, the other map (ore processing) has crusher hazards.